torch/_meta_registrations.py

```diff
@@ -1105,7 +1105,7 @@ def meta_relu_(self):
 
 @register_meta(aten.index_put.default)
 def meta_index_put(self, indices, values, accumulate=False):
-    return self.new_empty(self.size())
+    return torch.empty_like(self)
```

Comment:
we should audit the rest of the `new_empty` calls in this file..

Reply:
Yes. We should have instruction on which to use.... 
Stride behavior are subtle for these ops... 
torch.empty
torch.empty_like
torch.new_empty
torch.empty_strided